### DISCUSSION TITLE
Allow arbitrary breakpoint values for `hideBelow` and `hideFrom`

### DISCUSSION THREAD

**Discussion by hobbescodes** [chakra-ui/panda#1523] [Ideas]
So in its current state, `hideBelow` and `hideFrom` seem to only accept breakpoint tokens (i.e. `sm`, `md`, `lg`, etc). This is great for consistent design with what is defined in your config, however, there are issues that arise when having multiple components rendered in the same scope that use both `hideBelow` and `hideFrom`. The issue is that `hideBelow` and `hideFrom` are both inclusive on the designated breakpoint, therefore, at the exact point of said breakpoint, components will always be hidden. See: [Minimal reproducible example](https://play.panda-css.com/WqIPbM9KqD).

Proposed solution(s):

1. Redefine `hideBelow` to be noninclusive on provided breakpoints. The term "below" implies that it should not be hidden _at_ said breakpoint, but rather just hidden _up to_ said breakpoint.

2. If solution 1 is not conceivable (or possibly even it is), I would propose allowing arbitrary values (i.e. `700px`) to be passed to `hideBelow` and `hideFrom`. That way developers have more concise control over intended outcomes when using either `hideBelow` or `hideFrom`.

**Comment by FilipePfluck** [comment]
I agree `hideBelow` and `hideFrom` should be non inclusive. Then I suggest there could be `hideInAndBelow` and `hideInAndFrom` (or something like this), which are inclusive. 
And I agree it should be possible to use arbitrary values.
